userguide/tutorials/push_notifications.adoc

```diff
@@ -51,9 +51,77 @@ As explained earlier, once a handler is registered, each event is POST'ed to tha
 
 You can also read the https://docs.killbill.io/latest/kill_bill_events.html[Kill Bill Events] document to know more about events 
 
+== Testing steps:
+
+* For testing, run a local HTTP listener like the following:
+
+`docker run -p 8082:8080 -p 8443:8443 --rm -t mendhak/http-https-echo:28`
+
+* Register a notification handler:
+[source,bash]
+----
+curl -v \
+     -X POST \
+     -u admin:password \
+     -H "X-Killbill-ApiKey: bob" \
+     -H "X-Killbill-ApiSecret: lazar" \
+     -H "Content-Type: application/json" \
+     -H "Accept: application/json" \
+     -H "X-Killbill-CreatedBy: demo" \
+     -H "X-Killbill-Reason: demo" \
+     -H "X-Killbill-Comment: demo" \
+     "http://127.0.0.1:8081/1.0/kb/tenants/registerNotificationCallback?cb=http://localhost:8082/callback"
+----
+
+ * Make some configuration changes in a tenant so that Kill Bill pushes a notification on an event change. For example, https://docs.killbill.io/latest/quick_start_with_kaui.html#step2[add a new plan to a catalog].
```

Comment:
Nit: since all the other steps use `cURL` commands, maybe specify the equivalent `cURL` here instead of pointing to Kaui doc? Take a look at https://killbill.github.io/slate/#catalog-add-a-simple-plan.